darwin/index.js

```diff
@@ -1,9 +1,9 @@
 const electron = require('electron');
 const defaultMenu = require('electron-default-menu');
-
-const { NODE_ENV } = process.env;
+const isDev = require('electron-is-dev');
```

Comment:
Great find! We should add this to the other OSs

Reply:
I've been playing with this module and I don't really like it. When running with electron (not packaged) the default is always to set `isDev` to `true` which is annoying for anybody that doesn't have the local webpage server running on their machines.
If somebody wants to fork to fix a bug, we might end up with questions/issues that the code is not working because of this module.
Also, looking at their source code, this is so easy to implement that I think it'll be better to just read the environment variable ourselves instead of introducing a new dependency.
I think this single line can do the trick:
```js
const isDev = ('IS_DEV' in process.env) ? (process.defaultApp && process.env.IS_DEV === 'true') || false : false;
```

This will read `IS_DEV=true` (or whatever we want) and use that only if we're running under `electron` and not compiled. Anything else will always set it to `false`